main.tf

```diff
@@ -104,6 +104,15 @@ resource "aws_rds_cluster" "primary" {
     }
   }
 
+  dynamic "restore_to_point_in_time" {
+    for_each = var.clone_configuration
```

Comment:
Use upstream names to avoid confusion. In this "restore_to_point_in_time"

Reply:
good point